alf/environments/utils.py

```diff
@@ -145,7 +154,8 @@ def create_environment(env_name='CartPole-v0',
         env_load_fn = functools.partial(alf_wrappers.MultitaskWrapper.load,
                                         env_load_fn)
 
-    if hasattr(env_load_fn, 'batched') and env_load_fn.batched:
+    if hasattr(env_load_fn,
+               'batched') and env_load_fn.batched and batch_size_per_env == 1:
```

Comment:
need to update doc-string for batch_size_per_env

Reply:
Updated the doc-string.